spec/lib/soapy_cake/response_spec.rb

```diff
@@ -0,0 +1,15 @@
+RSpec.describe SoapyCake::Response do
+  subject { described_class.new('', '') }
+
+  describe '#parse_element' do
+    it 'converts keys ending in "_id" to integers' do
+      expect(subject.send(:parse_element, :conversion_id, '42')).to eq(42)
```

Comment:
Testing private method is generally considered bad practise. There are a couple of ways to do it better:

1. We just do integration tests with cases that cover the logic we implemented (good idea anyway, but usually not enough if we want to test thoroughly)
2. We test the public interface of the class (usually the best way, in this case it's a bit problematic because the class expects an XML doc as input which is bad to mock/fake)
3. We extract the parsing logic into a separate module/class and can properly unit-test that. 

If would probably go for 3. and also add an integration test that checks for the actual case you had. The best way would be to actually write this integration test first, make it fail and then try to solve the issue. That's usually faster than manually testing it over and over again.